core/core-spacemacs.el

```diff
@@ -82,6 +82,9 @@ the final step of executing code in `emacs-startup-hook'.")
   (setq dotspacemacs-editing-style (dotspacemacs//read-editing-style-config
                                     dotspacemacs-editing-style))
   (configuration-layer/initialize)
+  ;; frame title init
+  (when (and (display-graphic-p) dotspacemacs-frame-title-format)
+    (setq frame-title-format '((:eval (spacemacs/frame-title-prepare)))))
```

Comment:
I think that for tiling window managers the **icon-title-format** is more imporant than classic title because you can use it to differentiate between tabs.

As you can see on this screenshot, only active window have title and all others (where it would be really usefull) doesn't.

![Iconified Titles in Xmonad](https://www.dropbox.com/s/sikyjfdgfesybda/xmonad-iconified-titles.png?raw=1)

It make sense to have two special option for icon title as you may need much shorter version. I'm using layout where all tabs are equal sized, but  it may be more common to have active tab larger than non-active (at least icon name suggest this behavior).